transport/src/main/java/com/strategyobject/substrateclient/transport/ProviderInterface.java

```diff
@@ -41,6 +41,18 @@
      */
     Runnable on(ProviderInterfaceEmitted type, EventListener sub);
 
+    /**
+     * Send data to the node
+     *
+     * @param method      The RPC methods to execute
+     * @param params      Encoded parameters as applicable for the method
+     * @param isCacheable Request can be cached
+     * @return future containing result
+     */
+    CompletableFuture<Object> send(String method,
+                                   List<Object> params,
+                                   boolean isCacheable);
```

Comment:
The flag doesn't look good to me, I think, using a decorator is more preferable. IMHO, it isn't the responsibility of that interface/method.

Reply:
ok, let it be implemented on the caller side